### DISCUSSION TITLE
Table properties cannot be modified using ODBC connection

### DISCUSSION THREAD

**Discussion by lyh061619** [dbeaver/dbeaver#10638] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version 
Windows Server 2008 r2 Install Dict: D:\Program Files\dbeaver
- DBeaver version
version 7.3.0.202011291229
- Additional extensions
Navigator view :Advanced
#### Connection specification:
- Database name and version
Damon database（DM 7.6） URL:http://http://www.dameng.com/down.aspx?TypeId=11&FId=t14:11:14
- Driver name
Dm7JdbcDriver18
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
No SSH，URL: jdbc:dm://[IP]/[Database Instance]
#### Describe the problem you're observing:
Table properties cannot be modified
#### Steps to reproduce, if exist:
![图片](https://user-images.githubusercontent.com/15118571/101545298-07bab200-39e2-11eb-8108-2c0200efde73.png)
![图片](https://user-images.githubusercontent.com/15118571/101545329-11dcb080-39e2-11eb-9a0b-e9f41e84a840.png)

#### Include any warning/errors/backtraces from the logs
No

**Comment by kseniiaguzeeva** [comment, marked as answer]
We do not support this database that's why it is in read-only mode
